dspace/config/modules/requestitem.cfg

```diff
@@ -0,0 +1,43 @@
+#---------------------------------------------------------------#
+#----------------REQUEST ITEM CONFIGURATION---------------------#
+#---------------------------------------------------------------#
+
+# Configuration of request-item. Possible values:
+# all - Anonymous users can request an item
+# logged - Login is mandatory to request an item
+# empty/commented out - request-copy not allowed
+#request.item.type = all
+# Should all Request Copy emails go to the helpdesk instead of the item submitter?
+#request.item.helpdesk.override = false
+# Should a rejection of a copy request send an email back to the requester?
+# Defaults to "true", which means a rejection email is sent back.
+# Setting it to "false" results in a silent rejection.
+#request.item.reject.email = true
+# Bundles to use when granting access and inspecting file size, if "all files" is indicated
+# Default: ORIGINAL
+#request.item.grant.bundles = ORIGINAL
+# By default a granted request will send the file as an attachment.
+# If a secure web link to the bitstream(s) should be sent in some cases, enable the following property
+# Default: false
+#request.item.grant.link = true
+# If request.item.grant.link is enabled, you can specify a minimum file size (in megabytes) to use as
+# a threshold, or else the default attachment functionality will be used instead.
+# If 'all files' was indicated in the request, this threshold will be activated if any of the files
+# meet the configured size minimum.
+# To send links instead of attachments for all files, set this property to 0.
+# Default: 20
+#request.item.grant.link.filesize = 0
+# Valid access periods, in seconds, to allow the approver to select when using links.
+# These are presented to users with friendly labels via i18n keys in the angular frontend.
+# A 0 (or null) access period will be interpreted as 'forever'.
+# The first access period in the list will be the default option in the grant form dropdown.
```

Comment:
Are there any defaults?  If so, we should document the default values.  If there are no defaults, then we may want to uncomment the below list.  Sites may expect this to "just work" if they set `request.item.grant.link = true`.

Reply:
I've updated the defaults along with the storage period. they are uncommented since the feature is disabled by default so there is no harm keeping these open
```
request.item.grant.link.period = FOREVER
request.item.grant.link.period = +1DAY
request.item.grant.link.period = +1WEEK
request.item.grant.link.period = +1MONTH
request.item.grant.link.period = +3MONTHS
```